src/python/turicreate/toolkits/object_detector/object_detector.py

```diff
@@ -219,6 +219,9 @@ def create(dataset, annotations=None, feature=None, model='darknet-yolo',
     base_model = model.split('-', 1)[0]
     ref_model = _pre_trained_models.OBJECT_DETECTION_BASE_MODELS[base_model]()
 
+    pretrained_model = _pre_trained_models.OBJECT_DETECTION_BASE_MODELS['darknet_mlmodel']()
+    pretrained_model_path = pretrained_model.get_model_path()
```

Comment:
Doesn't this mean we're always downloading both the MXNet model and the Core ML model?

What gets printed to console when this code runs? It looks like we should be reaching this print statement twice: https://github.com/apple/turicreate/blob/4e191b87b028af7ae78a7d3491890a0887b9d013/src/python/turicreate/toolkits/_pre_trained_models.py#L62

Reply:
Yes we are downloading both MXNET and CoreML model and the print is executed twice (if the models are not stored in cache). I was wondering the same if we need both.

We use the darknet mxnet model in two places:
* to get `spatial_reduction` field - which we can add to the CoreML model init
* loading weights for the MXNET case when not using TF

Reply:
Okay, that makes sense as long as this is a transient state. I guess no matter what we need a later change to remove the old codepath.